src/nvim/api/vim.c

```diff
@@ -33,6 +34,22 @@
 # include "api/vim.c.generated.h"
 #endif
 
+/// @see foldCreate
```

Comment:
Thanks for the tip ! I've still a few things to do before passing this to RFC.
I am now looking at making the fold width autoadapt. I enable it with `set fdc=-1` to prevent the creation of a new variable. Is that ok ? If the PR gets too big, I could split it up anyway.